### PR TITLE
Documentation Suggestions

### PR DESCRIPTION
**Pull request by botanical**
What Changed:
- Made a couple documentation suggestions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,8 @@ make up-cumulus
 Since this will avoid deploying the other modules, deployment time will be
 shorter.
 
+One of the outputs from your deployment should be `cumulus_distribution_api_uri`. Copy the value to `.env` for `TF_VAR_cumulus_distribution_url`. Then redeploy.
```
Comment (botanical) [inline README.md:118]: For this particular addition, I'm wondering if adding some context here would be useful. I didn't add a cumulus doc link but can do that if it seems fitting

File: README.md [hunk 2]
```diff
@@ -157,12 +159,12 @@ update your Earthdata application as follows:
    **Manage > Redirect Uris**.
 1. Switch to your terminal window in your Docker container (if necessary, run
    `make docker`)
-1. Within your Docker container, run `make cumulus-tf/output` to show the
+1. Within your Docker container, run `make output-cumulus` to show the
```
Comment (botanical) [inline README.md:162]: I made this change because I didn't see any command mapping to `cumulus-tf/output` in the Makefile

**Review by chuckwondo — COMMENTED** [review]
Thanks for the addition. Just requesting a minor grammatical change and a formatting change (80-char line limit in `.md` files), but approving, since this is just a nit.

**Review by chuckwondo — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (chuckwondo) [reply README.md:163]: Ah yes! And I noticed another one, hang on...

Comment (chuckwondo) [reply README.md:163]: In `OPERATING.md`, line 194 (I think), `make docker` should be `make bash`

**Pull request merged**